src/sentry/api/endpoints/organization_integration_alert_rules.py

```diff
@@ -0,0 +1,32 @@
+from typing import Any
+
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization_integrations import RegionOrganizationIntegrationBaseEndpoint
+from sentry.integrations.opsgenie.integration import OpsgenieIntegration
+from sentry.models import Organization
+
+
+@region_silo_endpoint
+class OrganizationIntegrationAlertRulesEndpoint(RegionOrganizationIntegrationBaseEndpoint):
+    def put(
+        self,
+        request: Request,
+        organization: Organization,
+        integration_id: int,
+        **kwds: Any,
+    ) -> Response:
+        """
+        Migrate API keys and alert rules from plugin to integration
+        ```````````````````````````````````````````````````````````
+        :pparam string organization: the organization the integration is installed in
+        :pparam string integration_id: the id of the integration
+        """
+        integration = self.get_integration(organization.id, integration_id)
+        installation = integration.get_installation(organization_id=organization.id)
+        if isinstance(installation, OpsgenieIntegration):
```

Comment:
is this endpoint only for `Opsgenie`? the class name and the url path suggest this is a generic migration endpoint